ingestion/src/metadata/ingestion/connections/test_connections.py

```diff
@@ -289,7 +317,7 @@ def test_connection_db_common(
     service_connection,
     automation_workflow: Optional[AutomationWorkflow] = None,
     queries: dict = None,
-    timeout_seconds: int = 3 * 60,
+    timeout_seconds: Optional[int] = 3 * 60,
```

Comment:
make this a constant otherwise will be a pain to change when we need to

Reply:
was trying to avoid all the imports but yes agree, changed it